app/src/Routes/Works/Works.js

```diff
@@ -1,9 +1,26 @@
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
+import * as works from './works.json';
+
+const Piece = (piece) => (
+    <section>
```

Comment:
this maybe should be a `list` or something

Reply:
makes sense, done